ui/lib/passport-expiration.ts

```diff
@@ -4,7 +4,7 @@ import { useAccount } from "./use-wagmi";
 import { useVeNationLock } from "./ve-token";
 
 const decimalThreshold = Number(process.env.NEXT_PUBLIC_CITIZENSHIP_REVOKE_THRESHOLD)
```

Comment:
@southbears Is `NEXT_PUBLIC_CITIZENSHIP_REVOKE_THRESHOLD` a new environment variable?

Reply:
Based on this from the task description, revoke threshold seems like the most reasonable name:
```
Revoke threshold can be extracted fromrevokeUnderValue() on the passport contract or be hardcoded (by ENV variable) to 1.5 veNATION
```

Reply:
Technically the passport can be revoked under a threshold, but you can also estimate with the time function when the passport will "expire" (be able to be revoked). So for technical purposes, I think revoke is the most correct.